docs/source/install/index.rst

```diff
@@ -60,13 +60,38 @@ For Fedora/RHEL/CentOS users, run:
 
    sudo yum install -y gcc-c++ python3-devel make cmake
 
-Create conda environment for development, run:
+Create conda environment for development for CPU, run:
```

Comment:
`To create a Conda environment for CPU development, please execute the following command:`